bank-green/nuxt-bank-green
# Adding Cloudlfare Turntile Captcha to forms

## Changes
* using Cloudflare Turnstile instead of hCaptcha
* integrate actual server side validation of token -> `captcha-site-verify.post`
* integrate it on contact form as well
* replace Zapier links with env vars

## How to test
1. I've added localhost as valid domain in cloudflare but in order to not waste api calls to Turnstile, it's hidden on localhost. So if you wanna test it on localhost you need to remove the `v-if="!isLocal"`
2. You can go to /contact or /embrace and you should see the cloudflare checkbox. It will assess based on your browser and connection what verification you need to do. Usually it should just automatically give you a checkmark
